app/views/open_desk/central_navigation/menu.html.erb

```diff
@@ -1,20 +1,33 @@
 <%= turbo_frame_tag "opendesk-central-navigation" do %>
   <% @entries.each do |group| %>
-    <ul class="op-opendesk-navigation--group">
-      <li class="op-opendesk-navigation--item-header"><%= group.name %></li>
+    <%= render Primer::Alpha::ActionList.new(classes: "op-opendesk-navigation--group") do |list| %>
+      <% if group.name.present? %>
+        <% list.with_heading(
+             title: group.name,
+             align_items: :flex_start,
+             classes: "op-opendesk-navigation--item-header"
+           ) %>
+      <% end %>
+
       <% group.items.each do |item| %>
-        <li class="op-opendesk-navigation--item">
-          <%= link_to item.link, class: "op-opendesk-navigation--item-link op-menu--item-action" do%>
+        <% list.with_item(
+          href: item.link,
+          label: nil,
+          test_selector: "op-menu--item-action",
+          classes: "op-opendesk-navigation--item op-opendesk-navigation--item-link op-menu--item-action"
+        ) do |menu_item| %>
+          <div class="op-opendesk-navigation--item-content d-flex align-items-center gap-2">
```

Comment:
Please do not apply the Primer classes directly. We will for sure miss that when they are changed and the styles will break. Instead either apply the styles directly to the `op-opendesk-navigation--item-content` class or render a Primer Base component with tag `:div` and use the system arguments